util.c

```diff
@@ -1384,9 +1384,10 @@ char *Strcasestr(char *haystack, const char *needle)
 		lowneedle[i] = tolower(needle[i]);
 	ret = strstr(lowhay, lowneedle);
 	if (ret)
-    	return NULL;
-	ofs = ret - lowhay;
-    	return haystack + ofs;
+        return NULL;

```

Comment:
I think this is just some strangeness when I merged from your upstream repo.  I keep forgetting switch Xcode's indentation preferences when I edit cgminer code, so the indentations are different.  Otherwise these are the same.